processing/src/main/java/org/apache/carbondata/processing/merger/CarbonDataMergerUtil.java

```diff
@@ -351,6 +357,27 @@ public static boolean updateLoadMetadataWithMergeStatus(List<LoadMetadataDetails
           loadMetadataDetails.setMajorCompacted("true");
         }
 
+        if (DataMapUtil.isMVdatamapTable(carbonTable)) {
```

Comment:
I think it is not a good way to check whether it is MV table or not here. It should be checked in generic like it is child table or not? If it is child table get the extrainfo and fill it.